### PR TITLE
Clear up the verifying readme

### PR DESCRIPTION
**Pull request by ndorfin**
As a new player trying to verify my TA account, I came across some snags, which @mcoot helped me out with.
This PR is an attempt at clearing up the confusion around the Chat console vs. Unreal console.

### CODE REVIEW

File: docs/user_manual/getting_verified.md [hunk 1]
```diff
@@ -32,23 +32,21 @@ because that server is where most game servers are listed.
 Getting registered for the login server run by Griffon26 is an automated process:
 
 1. First log in to the Hirez servers
+1. Join a game server
+1. Open the Chat Console. (This might not have a keybinding, add one through `Settings > Keybindings > Interface`). Note: This console is different to the Unreal Console (~)
```
Comment (Griffon26) [inline docs/user_manual/getting_verified.md]: I think it's possible for anyone to open the chat console by pressing ESC, followed by t (assuming you haven't changed your chat key).

File: docs/user_manual/getting_verified.md [hunk 2]
```diff
@@ -32,23 +32,21 @@ because that server is where most game servers are listed.
...
+1. Send a private chat message to `@taserverbot` with only the word `authcode` in it:
 
-2. Send a private chat message to `taserverbot` with only the word authcode in it:
+    `@taserverbot authcode`
```
Comment (Griffon26) [inline docs/user_manual/getting_verified.md]: Technically you are sending a private message to taserverbot. The @ is how you send a private message, not who you send it to.
I assume that the "Say>" part caused confusion on whether you should type that yourself or not?
I think we should fix this all by using the technically accurate description that I had before, but then adding an animated screenshot of what it looks like to type this.

File: docs/user_manual/getting_verified.md [hunk 3]
```diff
@@ -32,23 +32,21 @@ because that server is where most game servers are listed.
...
-
-5. Fill in the authentication code and press OK
-
-6. If you don't get an error message, registration was successful.
-
-7. Restart Tribes Ascend and log in to Griffon26's server again to remove the "unvrf-" prefix.
+1. After logging in go to the "Store" menu and select "Redeem promotion"
+1. Fill in the authentication code and press OK
+1. If you don't get an error message, registration was successful.
+1. Restart Tribes Ascend and log in to Griffon26's server again to remove the "unvrf-" prefix.
```
Comment (Griffon26) [inline docs/user_manual/getting_verified.md]: I like to keep the source markdown as readable as possible by manually fixing the numbering and having empty lines between items. Could you restore this?

File: docs/user_manual/getting_verified.md (hunk 2 above)
Comment (ndorfin) [reply docs/user_manual/getting_verified.md]: Yeah, that was confusing. Hope the update covers that in more detail.

File: docs/user_manual/getting_verified.md (hunk 3 above)
Comment (ndorfin) [reply docs/user_manual/getting_verified.md]: Through past experience it's more difficult to edit fixed numbers, plus it also looks weird when it reaches 10 or more. Changed it back anyway.

File: docs/user_manual/getting_verified.md [hunk 4]
```diff
@@ -32,21 +32,33 @@ because that server is where most game servers are listed.
...
-1. Send a private chat message to `@taserverbot` with only the word `authcode` in it:
 
-    `@taserverbot authcode`
+2. Join a game server
 
-1. If you get an error saying the player taserverbot is not online, message Griffon26 in Discord
-1. Now log in to Griffon26's login server ("the community servers") with the in-game
+3. Open the Chat Console by pressing `T` (Default keybinding. If it's missing, add one through
+   `Settings > Keybindings > Interface`). Note: This chat console is different to the
+   Unreal Console (~)
```
Comment (Griffon26) [inline docs/user_manual/getting_verified.md]: They will have to press escape followed by `t`. `t` alone won't do the trick.
Write `t` in lower case so people don't think they have to use shift.
Remove the remark about keybinding or modify it to something like "If you have bound Chat to a different key than `t`, use that key instead".

**Comment by Griffon26** [comment]
I merged this by hand after applying the final changes.

**Pull request closed without merging**